site/specification/ORCv1.md

```diff
@@ -824,14 +824,31 @@ the index values and the additional value bits.
   bit is set, the entire value is negated.
 * Data values (W * L bits padded to the byte) - A sequence of W bit positive
   values that are added to the base value.
-* Patch list (PLL * (PGW + PW) bytes) - A list of patches for values
+* Patch list (PLL * ceil(PGW + PW) bits) - A list of patches for values
   that didn't fit within W bits. Each entry in the list consists of a
   gap, which is the number of elements skipped from the previous
   patch, and a patch value. Patches are applied by logically or'ing
   the data values with the relevant patch shifted W bits left. If a
   patch is 0, it was introduced to skip over more than 255 items. The
   combined length of each patch (PGW + PW) must be less or equal to
-  64.
+  64. (PGW + PW) is padded to the nearest fixed bit size according to the
+  below table before being encoded in the patch list.
+
+(PGW + PW)    | ceil(PGW + PW)
+:------------ | :-------------
+1 <= x <= 24  | x
+25            | 26
+26            | 26
+27            | 28
+28            | 28
+29            | 30
+30            | 30
+31            | 32
+32            | 32
+33 <= x <= 40 | 40
+41 <= x <= 48 | 48
+49 <= x <= 56 | 56
+57 <= x <= 64 | 64
```

Comment:
From

https://github.com/apache/orc/blob/9b79de995430b240cb68f225c22bef69ebbd8b07/c%2B%2B/src/RLEV2Util.cc#L29-L33

Reply:
Is it better to follow the java code to add 0 as well? https://github.com/apache/orc/blob/513922af6c1d1a07e632a03231d5b546eeafb26e/java/core/src/java/org/apache/orc/impl/SerializationUtils.java#L362-L391

Reply:
I don't think it makes sense to have 0 as the input is PGW + PW, and the spec states that PGW is 1 to 8 bits and PW is 1 to 64 bits, so it can never be 0 (or 1, for that matter, I guess :thinking: )